src/variables.js

```diff
@@ -9,6 +9,14 @@ if (!benefitservice_env_url && ENVIRONMENT !== "production") {
   benefitservice_env_url = "https://benefit-service-dev.dev.dts-stn.com";
 }
 
+// User Service URL
+let user_service_env_url = process.env.REACT_APP_USER_SERVICE_BASE_URL;
```

Comment:
need to add this to docker file and teamcity

Reply:
Dockerfile now has user service variables